### PR TITLE
Waste Reduction Logic App

### PR DESCRIPTION
**Pull request by ro100e**
## 🛠️ Description
Adding automation with the Waste Reduction Logic App is an automated detection mechanism provided by Azure Logic App The logic app will run on a configurable schedule to monitor selected subscriptions for a set of idle resources and send notifications once it finds any of those resources to alert admins to investigate and take action.

Added a new folder named logicapp within the docs folder and this is where the logic app deployment instructions are.
Added a new subfolder for media (.png files).
Added a new folder (waste reduction logic app) within the src folder.
Added to .json files for the logic app template code.

## 📷 Screenshots
Included in the deployment instructions located in the media folder

### 🔬 How did you test this change?

> - [x] 👍 Manually deployed + verified

### 🙋‍♀️ Do any of the following that apply?

> - [] 🚨 This is a breaking change.

### 📑 Did you update `docs/changelog.md`?

> - [x] ✅ Yes (required for `dev` PRs)

### 📖 Did you update documentation?

> - [x] ✅ Public docs in `docs` (required for `dev`)
> - [x] ✅ Internal dev docs in `src` (required for `dev`)

### CODE REVIEW

File: docs/_automation/logicApp/waste-reduction-logic-app.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+# Automated Detection
```
Comment (flanakin) [inline docs/_automation/logicApp/waste-reduction-logic-app.md:1]: Don't forget to add the page metadata like other doc pages.

**Review by flanakin — CHANGES_REQUESTED** [review]
I think we'll need a few PRs for this before it's ready for release. I created a new `features/alerts` branch. Can you resubmit this PR to that branch?

Once it's in a separate feature branch, my comments can be addressed in follow-up PRs. We just want to make sure that the only thing that goes into dev is complete, finalized changes including docs and changelog.

File: src/templates/waste reduction logic app/wasteApp.json
[the remark this answers is not in the document]
Comment (ro100e) [reply src/templates/waste reduction logic app/wasteApp.json:1]: Resolved

[the remark this answers is not in the document]
Comment (ro100e) [reply src/templates/waste reduction logic app/wasteApp.json:1]: Resolved

[the remark this answers is not in the document]
Comment (ro100e) [reply src/templates/waste reduction logic app/wasteApp.json:1]: Resolved

File: src/templates/waste reduction logic app/wasteApp-Gov.json
[the remark this answers is not in the document]
Comment (ro100e) [reply src/templates/waste reduction logic app/wasteApp-Gov.json:1]: Will add this to a new PR

File: docs/_automation/logicApp/waste-reduction-logic-app.md
[the remark this answers is not in the document]
Comment (ro100e) [reply docs/_automation/logicApp/waste-reduction-logic-app.md:1]: Will add this to a new PR

File: docs/_automation/logicApp/waste-reduction-logic-app.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  <img src="../logicapp/media/waste-app-2.png" alt="Showing the Edit template blade during the customer deployment process.">
+  <figcaption>
+    <center><strong>Figure WR-3</strong>. Edit template.</center>
+  </figcaption>
+</figure>
+
+> [!IMPORTANT]
+> The logic app needs to be in the same region as its resource group
+
+### Authorize Connection
```
[the remark this answers is not in the document]
Comment (ro100e) [reply docs/_automation/logicApp/waste-reduction-logic-app.md]: Will add this to a new PR

File: docs/_automation/logicApp/waste-reduction-logic-app.md (hunk 1 above)
Comment (ro100e) [reply docs/_automation/logicApp/waste-reduction-logic-app.md]: Will add this to a new PR

File: docs/_automation/logicApp/waste-reduction-logic-app.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+* Click on **Run Trigger** to test the Logic App ([Figure WR-15](../logicapp/media/waste-app-15.png))
+
+<figure>
+  <img src="../logicapp/media/waste-app-15.png" alt="Highlight the Run Trigger button in the Logic App designer tool">
+  <figcaption>
+    <center><strong>Figure WR-15</strong>. Run Trigger.</center>
+  </figcaption>
+</figure>
+
+<div style="text-align: right"> <h3>Next Up: <a href="../logicapp/acoi-reporting.md">Reporting</a></h2></div>
```
[the remark this answers is not in the document]
Comment (ro100e) [reply docs/_automation/logicApp/waste-reduction-logic-app.md]: Resolved

File: docs/_automation/logicApp/waste-reduction-logic-app.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+  <figure>
+    <img src="../logicapp/media/waste-app-6.png" alt="Showing the connected status for the WasteReductionConnection.">
+    <figcaption>
+      <center><strong>Figure WR-7</strong>. WasteReductionConnection.</center>
+    </figcaption>
+  </figure>
+
+### Configure Logic App
+
+After making sure the API Connection is connected, now you will need to create a system-assigned identity to allow the Logic App to "read" resources in the subscription.
```
[the remark this answers is not in the document]
Comment (ro100e) [reply docs/_automation/logicApp/waste-reduction-logic-app.md]: Will add this to a new PR

**Review by ro100e — COMMENTED** [review]
Items that have been resolved
- JSON-> Bicep
- Folder path (src/logic-apps)

File: src/logic-apps/waste-app.bicep [hunk 5]
```diff
@@ -0,0 +1,3639 @@
...
+            id: '${subscription().id}/providers/Microsoft.Web/locations/${WasteReductionLocation}/managedApis/office365'
+          }
+        }
+      }
+    }
+  }
+  name: WasteReductionName
+  location: WasteReductionLocation
+  tags: {
+    displayName: 'WasteReduction'
```
Comment (flanakin) [inline src/logic-apps/waste-app.bicep:3626]: Should this come from a parameter?

**Review by flanakin — APPROVED** [review]
@ro100e can you create an issue to track all the action items for future PRs? You can just create a simple task list in the issue so you can check them off as you go.
```markdown
- [ ] Task 1
- [ ] Task 2
- [ ] ...
```

File: src/logic-apps/waste-app.bicep [hunk 6]
```diff
@@ -0,0 +1,3639 @@
+param office365_1_Connection_Name string = 'WasteReductionConnection'
```
[the remark this answers is not in the document]
Comment (ro100e) [reply src/logic-apps/waste-app.bicep:1]: New PR will reflect this

File: src/logic-apps/waste-app.bicep [hunk 7]
```diff
@@ -0,0 +1,3639 @@
...
+param office365_1_Connection_DisplayName string = 'WasteReductionConnection'
```
[the remark this answers is not in the document]
Comment (ro100e) [reply src/logic-apps/waste-app.bicep:2]: `param ConnectionName string = 'WasteReductionConnection'
param DisplayName string = 'WasteReductionConnection'
`
This is what I have now, can make edits for future PRs if needed

**Pull request closed without merging**